thetis/options.py

```diff
@@ -245,6 +245,8 @@ def __init__(self):
 
         Bottom stress is :math:`\tau_b/\rho_0 = -g \mu^2 |\mathbf{u}|\mathbf{u}/H^{1/3}`
         """
+        self.wd_alpha = None
+        """Coefficient (units of length) used in bathymetry displacement function that ensures positive water depths."""
```

Comment:
Other options docstrings have the format `'python type: docstring'`, so this should be something like `Coefficient or None: Wetting-drying parameter...` Should also mention that the default is `None` which disables wetting-drying. Unit is meters.